src/apps/material/material.test.ts

```diff
@@ -183,6 +183,10 @@ describe("Material", () => {
 
     cy.getBySel("material-description").scrollIntoView();
 
+    // We need to wait here because no other fixes work.
+    // eslint-disable-next-line
+    cy.wait(1000);
```

Comment:
In this case, and the one below, do you know what we are waiting for?

Reply:
I think Sines question is relevant. Could you please document why the wait is necessary?